pkgs/development/libraries/glibc/common.nix

```diff
@@ -191,7 +190,7 @@ stdenv.mkDerivation ({
     libc_cv_c_cleanup=yes
     libc_cv_gnu89_inline=yes
     # Only due to a problem in gcc configure scripts:
-    libc_cv_sparc64_tls=${if cross.withTLS then "yes" else "no"}
+    libc_cv_sparc64_tls=yes
```

Comment:
I think this is unused now as well.

Reply:
Oh figured as much and was going to try to make this not a cross mass rebuild, but it is already. Good catch.